examples/model_compress/model_speedup.py

```diff
@@ -9,145 +10,89 @@
-        return
-
-def slim_speedup(masks_file, model_checkpoint):
-    device = torch.device('cuda')
-    model = VGG(depth=19)
-    model.to(device)
-    model.eval()
-
-    dummy_input = torch.randn(64, 3, 32, 32)
-    if use_mask:
-        apply_compression_results(model, masks_file)
-        dummy_input = dummy_input.to(device)
-        start = time.time()
-        for _ in range(32):
-            out = model(dummy_input)
-        #print(out.size(), out)
-        print('mask elapsed time: ', time.time() - start)
-        return
-    else:
-        #print("model before: ", model)
-        m_speedup = ModelSpeedup(model, dummy_input.to(device), masks_file)
-        m_speedup.speedup_model()
-        #print("model after: ", model)
-        dummy_input = dummy_input.to(device)
-        start = time.time()
-        for _ in range(32):
-            out = model(dummy_input)
-        #print(out.size(), out)
-        print('speedup elapsed time: ', time.time() - start)
-        return
+            use_speedup_out = model(dummy_input)
+        print('elapsed time when use speedup: ', time.time() - start)
+    if compare_results:
+        if torch.allclose(use_mask_out, use_speedup_out, atol=1e-07):
+            print('the outputs from use_mask and use_speedup are the same')
+        else:
+            raise RuntimeError('the outputs from use_mask and use_speedup are different')
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser("speedup")
     parser.add_argument("--example_name", type=str, default="slim", help="the name of pruning example")
     parser.add_argument("--masks_file", type=str, default=None, help="the path of the masks file")
-    parser.add_argument("--model_checkpoint", type=str, default=None, help="the path of checkpointed model")
     args = parser.parse_args()
-    
-    if args.example_name == 'slim':
-        if args.masks_file is None:
-            args.masks_file = 'mask_vgg19_cifar10.pth'
-        slim_speedup(args.masks_file, args.model_checkpoint)
-    elif args.example_name == 'fpgm':
-        if args.masks_file is None:
-            args.masks_file = 'mask.pth'
-        fpgm_speedup(args.masks_file, args.model_checkpoint)
-    elif args.example_name == 'l1filter':
-        if args.masks_file is None:
-            args.masks_file = 'mask_vgg16_cifar10.pth'
-        l1filter_speedup(args.masks_file, args.model_checkpoint)
-    elif args.example_name == 'apoz':
-        if args.masks_file is None:
-            args.masks_file = 'mask_vgg16_cifar10.pth'
-        apoz_speedup(args.masks_file, args.model_checkpoint)
+
+    if args.example_name != 'all':
+        if args.masks_file is not None:
+            config[args.example_name]['masks_file'] = args.masks_file
+        if not os.path.exists(config[args.example_name]['masks_file']):
+            msg = '{} does not exist! You should specify masks_file correctly, '
+            msg += 'or use default one which is generated by model_prune_torch.py'
```

Comment:
Use multi-line string. `+=` is not necessary.

Reply:
fixed, thanks!